package.json

```diff
@@ -42,6 +42,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hot-toast": "^2.4.1",
+    "react-icons": "^4.10.1",
```

Comment:
We have MUI icons installed. Try to see if you can use those before adding another icon package.